### ISSUE TITLE
publish-listing goal fails when title.txt is exactly 30 characters

### ISSUE THREAD

**Issue by athkalia** [simpligility/android-maven-plugin#698]
When I run the publish-listing goal with a title that has exactly 30 characters then I get the error message:

" Problem in the listing content: Too many characters in file title.txt max allowed is 30 "

I think that the Problem lies in https://github.com/simpligility/android-maven-plugin/blob/master/src/main/java/com/simpligility/maven/plugins/android/AbstractPublisherMojo.java

```
 while ( line != null )
            {
                sb.append( line );
                sb.append( "\n" );
                line = br.readLine();
            }
            everything = sb.toString();
```
where you are appending into a StringBuilder the line separator. If the title.txt is already exactly 30 chars in a single line then your count that includes the line separator will show 31, resulting in the described error.

Cheers,
Sakis

**Comment by Shusshu** [comment]
Would you like to provide a pull request which fixes the issue?

**Comment by athkalia** [comment]
Gladly. It's going to be around next weekend.

**Comment by lucasiturbide** [comment]
@athkalia I think this is happening for the shortdescription as well.  I didn't see any reported issue with that. Is there a chance to fix that as well?

**Comment by athkalia** [comment]
Yeah, sure. The change is in the abstractPublisherMojo so it affects both descriptions and titles

**Comment by athkalia** [comment]
Created pull request here https://github.com/simpligility/android-maven-plugin/pull/703

**Comment by Shusshu** [comment]
Please test with latest code

**Comment by mosabua** [comment]
Should be fixed with 4.4.1 release. Please open new issue if not.
